Jellyfin.Server/Migrations/MigrationRunner.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Linq;
+using MediaBrowser.Common.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Server.Migrations
+{
+    /// <summary>
+    /// The class that knows how migrate between different Jellyfin versions.
+    /// </summary>
+    public sealed class MigrationRunner
+    {
+        /// <summary>
+        /// The list of known migrations, in order of applicability.
+        /// </summary>
+        internal static readonly IUpdater[] Migrations =
+        {
+            new Routines.DisableTranscodingThrottling(),
+            new Routines.DisableZealousLogging()
+        };
+
+        /// <summary>
+        /// Run all needed migrations.
+        /// </summary>
+        /// <param name="host">CoreAppHost that hosts current version.</param>
+        /// <param name="loggerFactory">Factory for making the logger.</param>
+        public static void Run(CoreAppHost host, ILoggerFactory loggerFactory)
+        {
+            var logger = loggerFactory.CreateLogger<MigrationRunner>();
+            var migrationOptions = ((IConfigurationManager)host.ServerConfigurationManager).GetConfiguration<MigrationOptions>(MigrationsListStore.StoreKey);
+            var applied = migrationOptions.Applied.ToList();
```

Comment:
`ToHashSet()` would be more appropriate here

Reply:
That was my initial implementation, but I cannot maintain the order in the stored list.
Not that it matters currently, but keeping the order of applied routines on disk might get useful in the future, and this speed improvement is a one-time and negligible at best :)